aggregator/src/datastore.rs

```diff
@@ -4217,11 +4176,11 @@ pub mod models {
         }
     }
 
-    /// AcquiredCollectJob represents an incomplete collect job whose lease has been acquired.
+    /// AcquiredCollectJob represents an incomplete collection job whose lease has been acquired.
     #[derive(Clone, Debug, PartialOrd, Ord, Eq, PartialEq)]
     pub struct AcquiredCollectJob {
```

Comment:
nit: `AcquiredCollectJob` ->  `AcquiredCollectionJob`?

(and similarly below, `CollectJob`, `CollectJobState`, & perhaps others should be updated. I think `s/CollectJob/CollectionJob/g` would be a good change to make, too.)